ui/pages/send/send-content/add-recipient/add-recipient.component.test.js

```diff
@@ -178,29 +152,30 @@ describe('AddRecipient Component', () => {
     it('should render error when query has ens does not resolve', () => {
       wrapper.setProps({
         addressBook: [],
-        toError: 'bad',
-        ensResolutionError: 'very bad',
+        ensError: 'very bad',
         contacts: [],
         nonContacts: [],
       });
 
       const dialog = wrapper.find(Dialog);
 
       expect(dialog.props().type).toStrictEqual('error');
-      expect(dialog.props().children).toStrictEqual('very bad');
+      expect(dialog.props().children).toStrictEqual('very bad_t');
```

Comment:
What's the extra `_t`?

Reply:
Some errors from ENS previously weren't translated, this is expanded now and this section under test was impacted

Reply:
but I did catch two new error cases that were added in this PR that weren't translated yet and added them. Thanks @darkwing